data/DataType/bool_title_case.yaml

```diff
@@ -0,0 +1,7 @@
+keyword: bool
+integer-min: '0'
+integer-max: '1'
+integer-signed: false
+literals:
+  - 'True'
+  - 'False'
```

Comment:
no new line character at the end of file (new-line-at-end-of-file)

*Origin: YAMLLintBear, Section: `yaml`.*